### ISSUE TITLE
Router: Log router steps needed to find right action step

### ISSUE THREAD

**Issue by tippexs** [nginx/unit#758]
It would be very helpful to log the steps the Unit router takes to find the correct route for an incoming request to the `unit.log`.

The idea behind this is similar to the idea of NGINX creating rewrite logs. http://nginx.org/en/docs/http/ngx_http_rewrite_module.html#rewrite_log

This issue is open for discussions and other ideas. The final output could be similar to the NGINX log output.

**Comment by mwoodpatrick** [comment]
One thing I was missing was the trailing "/*" in the URI        
     ```
   {
            "match": {
                "uri": "/example_php/*"
            },            
            "action": {
                "pass": "applications/example_php"
            }
        }
```

Without it the access appears to be silently ignored with it the access is attempted and an error message describing the issue is reported

**Comment by lcrilly** [comment]
Router diagnostic logging shipped with unit-1.30.0
